doc/sphinx/addendum/micromega.rst

```diff
@@ -288,48 +298,89 @@ obtain :math:`-1`. By Theorem :ref:`Psatz <psatz_thm>`, the goal is valid.
    The :tacn:`zify` tactic can be extended with new types and operators by declaring and registering new typeclass instances using the following commands.
    The typeclass declarations can be found in the module ``ZifyClasses`` and the default instances can be found in the module ``ZifyInst``.
 
-.. cmd:: Add Zify {| InjTyp | BinOp | UnOp |CstOp | BinRel | UnOpSpec | BinOpSpec } @qualid
+.. cmd:: Add Zify @add_zify @one_term
+
+   .. insertprodn add_zify add_zify
+
+   .. prodn::
+      add_zify ::= {| InjTyp | BinOp | UnOp | CstOp | BinRel | UnOpSpec | BinOpSpec }
+      | {| PropOp | PropBinOp | PropUOp | Saturate }
+
+   Registers an instance of the specified typeclass.
 
-   This command registers an instance of one of the typeclasses among ``InjTyp``, ``BinOp``, ``UnOp``, ``CstOp``,  ``BinRel``,
-   ``UnOpSpec``, ``BinOpSpec``.
+.. cmd:: Show Zify @show_zify
 
-.. cmd:: Show Zify {| InjTyp | BinOp | UnOp |CstOp | BinRel | UnOpSpec | BinOpSpec }
+   .. insertprodn show_zify show_zify
 
-   The command prints the typeclass instances of one the typeclasses
-   among ``InjTyp``, ``BinOp``, ``UnOp``, ``CstOp``, ``BinRel``,
-   ``UnOpSpec``, ``BinOpSpec``. For instance, :cmd:`Show Zify` ``InjTyp``
+   .. prodn::
+      show_zify ::= {| InjTyp | BinOp | UnOp | CstOp | BinRel | UnOpSpec | BinOpSpec | Spec }
+
+   .. todo PR: should there be Show Zify {| PropBinOp | PropOp | PropUOp | Saturate } options?
+      to match Add Zify ...   An oversight or not?
```

Comment:
What do you mean by options? Things you can set?
`Show Zify bla`  does not enable printing. It prints information when invoked. 

I hope it answers the question.

Reply:
Perhaps I should have said "commands" instead of "options".  For symmetry with `Add Zify`, should `Show Zify` also support `PropOp | PropBinOp | PropUOp | Saturate`?  If so, I will add them in another PR.

![image](https://user-images.githubusercontent.com/1253341/96506356-53d04e00-120c-11eb-9794-7d680c7d7563.png)

Reply:
@fajb says:

> As the set of such operators looks fixed to me.
I'll rather remove the `Add PropOp` PropBinOp et PropUOp.
That is convenient for implementation but the user probably does not
care.